adap/htdocs/register.php

```diff
@@ -0,0 +1,134 @@
+<?php
+  include 'homepage/navbar_before_login.php';
+?>
+<!DOCTYPE html>
+<head>
+  <title>ADAP</title>
+  <!--Import Google Icon Font-->
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <!--Import materialize.css-->
+  <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
```

Comment:
It's super puzzling but somehow when I inspect this shows up as a 404 error 🤔 All the other links for javascript or css files as well. Is it possible to bundle your `php` files into a single folder instead? 🤔